### PR TITLE
Fix keyboard keys again

### PR DESCRIPTION
**Pull request by TotalCaesar659**
https://ru.m.wikipedia.org/wiki/Print_Screen

### CODE REVIEW

File: wikipedia-dark.user.css [hunk 1]
```diff
@@ -4558,10 +4558,11 @@ regexp("https?:\/\/([\w\-]{2,}\.m\.(wiki(pedia|books|news|quote|source|versity|v
   .client-js body.mw-rcfilters-ui-loading .mw-changeslist {
     opacity: .7;
   }
-  kbd.keyboard-key {
+  kbd.keyboard-key, kbd {
     background: var(--gray-18) !important;
     border-color: var(--gray-5) !important;
     color: var(--gray-9a) !important;
+    -webkit-box-shadow: .1em .2em .2em var(--gray-4) !important;
```
Comment (Mottie) [inline wikipedia-dark.user.css:4565]: Please change this to `box-shadow` so it works in all browsers

**Review by Mottie — CHANGES_REQUESTED** [review]

File: wikipedia-dark.user.css (hunk 1 above)
Comment (the-j0k3r) [reply wikipedia-dark.user.css:4565]: also make ru changes into ru section. other section may not reach this and vice versa

Comment (TotalCaesar659) [reply wikipedia-dark.user.css:4565]: Where I need to move it? Tried section ru.wikipedia.org (line 6307) but it doesn't apply.

Comment (the-j0k3r) [reply wikipedia-dark.user.css:4565]: its because youre modding ru mobile, so create one?

Comment (TotalCaesar659) [reply wikipedia-dark.user.css]: Got it.

**Pull request merged**